tests/test_germline_dna.yml

```diff
@@ -49,5 +49,5 @@
     - path: "test-output/multisample_variants/multisample.vcf.gz.tbi"
     - path: "test-output/multisample_variants/stats"
     - path: "test-output/samples/NA12878/lib_U0a/NA12878-U0a.markdup.bam"
-    - path: "test-output/samples/NA12878/lib_U0a/NA12878-U0a.markdup.bai"
```

Comment:
Why don't we check for the index anymore?

Reply:
I worked on an outdated branch by accident. The merging caused some issues. Fixed this